charts/cp-schema-registry/templates/schema-registry-ui.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  {{- if .Values.ui.annotations }}
+  annotations: {{ toYaml .Values.ui.annotations | nindent 4 }}
+  {{- end}}
+  labels:
+    app: {{ template "cp-schema-registry.fullname" . }}
+  {{- if .Values.ui.labels }}
```

Comment:
This indentation of this `if` (and the `end`) looks strange to me.
Because you're adding items inside the labels block, not below it

Shouldn't it be 2 spaces more indented, so like this:
```
  labels:
    app: {{ template "cp-schema-registry.fullname" . }}
    {{- if .Values.ui.labels }}
{{ toYaml .Values.ui.labels | indent 4}}
    {{- end}}
  name: {{ template "cp-schema-registry.fullname" . }}-ui
```
?